bicep/files-to-load/create_cc_param.py

```diff
@@ -99,15 +99,140 @@ def set_params(params, dbPassword, outputs):
         params['AdditionalNFSMountOptions'] = outputs['filerInfoFinal']['value']['additional']['mountOptions']
         params['AdditionalNFSAddress'] = outputs['filerInfoFinal']['value']['additional']['ipAddress']
 
+
+def set_ood_params(params, outputs):
+    slurm_params = get_json_dict('initial_params.json')
+    # We want to essentially inherit certain settings from the slurm cluster.
+    set_slurm_params(slurm_params, "", outputs)
+    params['NFSAddress'] = slurm_params.get('NFSAddress') or 'ccw-scheduler'
+    params['NFSSharedExportPath'] = slurm_params.get('NFSSharedExportPath') or '/shared'
+    params['NFSSharedMountOptions'] = slurm_params.get('NFSSharedMountOptions')
+    params['SubnetId'] = slurm_params["SubnetId"]
+    params['Region'] = slurm_params['Region']
+    params['Credentials'] = slurm_params['Credentials']
+
+    params['MachineType'] = outputs['ood']['value'].get('sku')
+    params['ManagedIdentity'] = outputs['ood']['value'].get('managed_identity')
+    params['BootDiskSize'] = outputs['ood']['value'].get('BootDiskSize')
+    params['ImageName'] = outputs['ood']['value'].get('ImageName')
+
+    params['ood_server_name'] = outputs['ood']['value'].get('ood_server_name')
+    params['ood_auth_method'] = outputs['ood']['value'].get('ood_auth_method')

```

Comment:
Please check the new template as LDAP and auth method has been removed, but a NIC Id added